docs/api.md

```diff
@@ -218,10 +360,13 @@ From the CLI, one can either:
    configuration options. The VM can then be asynchronously created and booted
    by sending HTTP commands to the [REST API](#rest-api). Check the
    [REST API examples](#rest-api-examples) section for more details.
+1. Start the [D-Bus API](#d-bus-api), by not passing any VM
```

Comment:
Nit: I think we can combine it with the HTTP paragraph above and mention using HTTP API and D-Bus API at the same time is also allowed. 

Also, you may forget to add the D-Bus bit to the content between line 355 and line 358.

Reply:
> Also, you may forget to add the D-Bus bit to the content between line 355 and line 358.

I completely overlooked changing this section, thanks! Made the changes accordingly, PTAL.